packages/mc-scripts/src/utils/get-config-diff.spec.js

```diff
@@ -0,0 +1,224 @@
+const getConfigDiff = require('./get-config-diff');
+
+const oldConfig = {
+  name: 'test name',
+  description: 'test description',
+  url: 'https://test.com',
+  icon: '<svg><path fill="#000000"></path></svg>',
+  permissions: [],
+  mainMenuLink: {},
+  submenuLinks: [],
+};
+
+const newConfig = {
+  name: 'test name',
+  description: 'test description',
+  url: 'https://test.com',
+  icon: '<svg><path fill="#000000"></path></svg>',
+  permissions: [],
+  mainMenuLink: {},
+  submenuLinks: [],
+};
```

Comment:
I would suggest to define a function to return an object

```js
const createTestConfig = (custom) => ({
  name: 'test name',
  // ...
  ...custom,
})
```

Then you can use the function to initialize e.g. old/new config values etc.